### PR TITLE
ZCS-2188 documenting ephemeral backup/restore from SSDB

### PR DESCRIPTION
**Pull request by tmclane**
Initial pass at documenting SSDB backup restore.
Wanted to get some feedback and see which direction is desired.

### CODE REVIEW

File: backuprestore.adoc [hunk 1]
```diff
@@ -1553,14 +1553,80 @@ auth tokens generated prior to the backup will no longer work.
 The Jira ticket ZMS-614 has been opened to track the potential future development
 of ephemeral data backups.
 
-=== Backups with SSDB Backend
+=== SSDB Backend
```
Comment (fatkudu) [inline backuprestore.adoc]: Maybe *Ephemeral store SSDB backend*

**Comment by tmclane** [comment]
As long as the data stored in SSDB doesn't change from one version of the mailbox to another I don't see how a restore would cause any incompatibility.
The data format doesn't change in SSDB and it is accessed through the SSDB / Redis protocol which  insulates the client from the actual in-memory storage scheme.

**Comment by tmclane** [comment]
@fatkudu There is already a section entitled `SSDB Configuration Options` in the Admin Guide that contains the information on how to configure SSDB in it's various HA modes.
Surely you don't want to duplicate that information?

File: backuprestore.adoc (hunk 1 above)
Comment (tmclane) [reply backuprestore.adoc]: Isn't this clear enough considering it's under the heading of `Notes on Ephemeral Data`?

Comment (tmclane) [reply backuprestore.adoc]: I can prefix it if you like however.

**Comment by fatkudu** [comment]
@tmclane duplication is indeed bad but links to original information can often be useful.

File: backuprestore.adoc (hunk 1 above)
Comment (fatkudu) [reply backuprestore.adoc]: Not hugely fussed.

**Pull request merged**